src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -127,8 +127,8 @@ private void OnBuiltCore(DistributedApplication application)
     private void OnBuilderCreatingCore(DistributedApplicationOptions applicationOptions, HostApplicationBuilderSettings hostBuilderOptions)
     {
         hostBuilderOptions.EnvironmentName = Environments.Development;
-        hostBuilderOptions.ApplicationName = typeof(TEntryPoint).Assembly.GetName().Name ?? string.Empty;
-        applicationOptions.AssemblyName = typeof(TEntryPoint).Assembly.GetName().Name ?? string.Empty;
+        hostBuilderOptions.ApplicationName = entryPoint.Assembly.GetName().Name ?? string.Empty;
+        applicationOptions.AssemblyName = entryPoint.Assembly.GetName().Name ?? string.Empty;
```

Comment:
```suggestion
        hostBuilderOptions.ApplicationName = _entryPoint.Assembly.GetName().Name ?? string.Empty;
        applicationOptions.AssemblyName = _entryPoint.Assembly.GetName().Name ?? string.Empty;
```

Should use the field everywhere.

Reply:
I'm just talking about in this private instance method.

Reply:
"Should use the field everywhere _possible_"